### PR TITLE
Solution 

### PR DESCRIPTION
**Pull request by W2AlharbiMe**
How difficult did you find this (0 - 10)?
1
How long did it take?
30 minutes
Was there anything that you struggled with? If so, what?
no 
Is there anything that you would like some further information on?
no
Do you have any suggestions to improve this assignment?
no

### CODE REVIEW

File: dice.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const generateNumber = function(n) {
+  n = n ? n : 6;
```
Comment (Salman778) [inline dice.js:2]: Great solution; Also you can allow named parameters to be initialized with default values if no value or undefined is passed:
```const generateNumber = function(n=6){...}```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters

File: dice.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  return Math.round(Math.random() * n);
```
Comment (Salman778) [inline dice.js:3]: Here you return value within a range starting from 0 to n; but as we know the dice has 6 values starting from 1 to 6 so we need to enhance the range.
```Math.round(Math.random() * (max - min) + min);```

**Comment by Salman778** [comment]
You are doing a good job! Keep going!

File: dice.js (hunk 2 above)
Comment (W2AlharbiMe) [reply dice.js:3]: thank you for your feedback, i'll work on the enhancements after my exam ان شاء الله , have a nice day

**Pull request closed without merging**